src/components/DataInput/DataInput.cy.tsx

```diff
@@ -40,6 +41,13 @@ describe("Data Input Components", () => {
                 onChange={getFreeFormTextHandler(() => console.log("Freeform Changed"))}
             />
         );
+        cy.mount(
+            <PasswordInput
+                label="Password"
+                defaultValue="password"
+                onChange={getFreeFormTextHandler(() => console.log("Password Chanegd"))}
```

Comment:
Changed*

Reply:
This actually isn't a mistake, it's derived from the old latin for the infinitive "to change"

...no I'm just kidding, that's a good spot